src/extension-runners/firefox-android.js

```diff
@@ -411,17 +411,34 @@ export class FirefoxAndroidExtensionRunner {
     log.debug('Checking read/write permissions needed for web-ext' +
               `on ${selectedFirefoxApk}...`);
 
-    // Runtime permission needed to be able to run Firefox on a temporarily created profile
-    // on android versions >= 23 (Android Marshmallow, which is the first version where
-    // these permissions are optional and have to be granted explicitly).
+    // Runtime permissions needed to Firefox to be able to access the
+    // xpi file uploaded to the android device or emulator.
     const requiredPermissions = [
       'android.permission.READ_EXTERNAL_STORAGE',
-      'android.permission.WRITE_EXTERNAL_STORAGE',
     ];
 
     await adbUtils.ensureRequiredAPKRuntimePermissions(
       selectedAdbDevice, selectedFirefoxApk, requiredPermissions
     );
+
+    // Runtime permission needed to be able to run Firefox on a temporarily created profile
```

Comment:
> Is this really needed? The profile directory is now created in `/data/local/tmp/` instead of `/sdcard/`: https://github.com/mozilla/web-ext/blame/c884696fd3ea496c67959337f98d4e39bdb08871/src/util/adb.js#L14

I know that ;-), and I also described it explicitly in the pull request description.

Personally I'm not really against removing it all together, the only reason I left it as a debug message is that we still have docs related to Fennec (e.g. [in extensionworkshop.com](https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android-fennec/), as you already know), but even removing it wouldn't prefer a user to use web-ext with Fennec (the extension developer would need to be sure to add both those prefs).

I'll update the PR to remove it (it may also be enough to bring back the coverage to the previous threshold and make codecov happy again on this PR).